package.xml

```diff
@@ -21,6 +21,7 @@
   <depend>rclcpp</depend>
   <depend>rclcpp_lifecycle</depend>
   <depend>rclcpp_components</depend>
+  <depend>ros_environment</depend>
```

Comment:
I think we can skip this dependency if we use rclcpp_VERSION_MAJOR in the cmakelists.